openpype/pipeline/workfile/workfile_template_builder.py

```diff
@@ -1431,8 +1455,12 @@ def _get_representations(self, placeholder):
                 "representation": [placeholder.data["representation"]],
                 "family": [placeholder.data["family"]],
             }
-
         else:
+            if builder_type != "all_assets":
+                print(97, self)
+                print(98, builder_type)
+                self.project_name = builder_type
```

Comment:
I'm really not sure if it's good idea to change project of workfile builder. Maybe the project name should be defined on placeholder rather than change dynamically value of `project_name` on objects (and this machinery).

In that case it would be more bullet proof, as each placeholder have the information, and rest of logic can avoid using the project from workfile builder.